### PR TITLE
Unignore one MethodConfigSpec UI test - No JIRA ticket

### PR DESCRIPTION
**Pull request by aweng98**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by aweng98** [comment]
https://broadinstitute.atlassian.net/browse/GAWB-3245

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala [hunk 1]
```diff
@@ -206,15 +206,21 @@ class MethodConfigSpec extends FreeSpec with WebBrowserSpec with CleanUp with Wo
...
       withSignIn(user) { _ =>
         val workspaceMethodConfigPage = new WorkspaceMethodConfigListPage(billingProject, workspaceName).open
-        val methodConfigDetailsPage = workspaceMethodConfigPage.importMethodConfigFromRepo(MethodData.SimpleMethod.methodNamespace,
-          MethodData.SimpleMethod.methodName, MethodData.SimpleMethod.snapshotId, methodName, Some(MethodData.SimpleMethod.rootEntityType))
+
+        val methodConfigDetailsPage = workspaceMethodConfigPage.importMethodConfigFromRepo(
+          MethodData.SimpleMethod.methodNamespace,
+          MethodData.SimpleMethod.methodName,
+          MethodData.SimpleMethod.snapshotId,
+          SimpleMethodConfig.configName,
```
Comment (breilly2) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodConfigSpec.scala:221]: Ah, so this was the cause of the problem. Makes sense. Was probably a mistake in a mechanical refactoring and we had bigger problems at the time than to fix it.

**Review by breilly2 — APPROVED** [review]
The changes to `Component.isVisible` and (to a lesser extent) `WorkspaceDataPage.awaitReady` are much further reaching than the test being ignored so I want to see the full suite pass. As long as it does, 👍

**Review by andy7i — APPROVED** [review]
👍

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Comment by aweng98** [comment]
jenkins retest

**Pull request merged**